controllers/operands/kubevirtConsolePlugin.go

```diff
@@ -119,6 +119,7 @@ func NewKvUiPluginDeplymnt(hc *hcov1beta1.HyperConverged) (*appsv1.Deployment, e
 							},
 						},
 					},
+					NodeSelector:      hc.Spec.Infra.NodePlacement.NodeSelector,
```

Comment:
This line fails on unit test. Please first check that `c.Spec.Infra.NodePlacement` is not nil.

Reply:
I don't understand how that's related to the test failures we're seeing (https://github.com/kubevirt/hyperconverged-cluster-operator/pull/2310#issuecomment-1521153623). Anyway, I have cherry-picked commits from #2319 that address the `nil` check and also add tests. Running tests locally still causes similar error.